trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk

```diff
@@ -154,11 +155,20 @@
 {% set paymentDetailsHtml %}
   <h2 class="govuk-heading-l" data-cy="payment-details-heading">Payment details</h2>
 
-  <p class="govuk-body">
-    {% if paymentDetails.length === 0 %}
-      Payment details will be displayed when payments have been entered on the premium payments tab.
-    {% endif %}
-  </p>
+  {% if paymentDetails.length === 0 %}
+    <p class="govuk-body">Payment details will be displayed when payments have been entered on the premium payments tab.</p>
+  {% else %}
+    {{ govukButton({
+      text: "Show filter",
+      classes: "govuk-button govuk-button--secondary govuk-!-margin-left-2",
+      id: "payment-details-show-filter-button",
+      attributes: {'data-cy': 'payment-details-show-filter-button'}
+    }) }}
```

Comment:
Is this just a placeholder at the minute? or am I misunderstanding the filtering system? Cheers

Reply:
Yes it is - the button will be implemented as part of [FN-2311](https://ukef-dtfs.atlassian.net/browse/FN-2311)